fuuk/people/views.py

```diff
@@ -17,7 +16,7 @@ class ArticleList(ListView):
 
     def get_queryset(self):
         queryset = Article.objects.filter(type__in=('ARTICLE', 'BOOK'))
-        self.years = queryset.values_list('year', flat=True).annotate(Count('year')).order_by('-year')
+        self.years = queryset.values_list('year', flat=True).distinct().order_by('-year')
```

Comment:
Are you sure about that. It seems to me, that it is the other way as the effectiveness is concerned, but it would require some profiling.